packages/icons/package.json

```diff
@@ -20,11 +20,6 @@
         "dependsOn": [
           "compile"
         ]
```

Comment:
question: doesn't need to be changed [here](https://github.com/juntossomosmais/atomium/blob/main/packages/core/package.json#L40-L62) and [here](https://github.com/juntossomosmais/atomium/blob/main/apps/docs/package.json#L8-L31) too?

Reply:
Yes, it will also be necessary too

Reply:
thanks a lot guys! done at [d0cbd19](https://github.com/juntossomosmais/atomium/pull/289/commits/d0cbd19fbda0e24ca58f7732ec0c2cb8d2371870) and [7f1214f](https://github.com/juntossomosmais/atomium/pull/289/commits/7f1214fdd08a98797946e3b2fcc99e7eea04f26e)